tests/kuttl/test-suites/default/watcher-api-scaling/01-assert.yaml

```diff
@@ -328,6 +336,76 @@ metadata:
 spec:
   serviceName: watcher
 ---
+apiVersion: watcher.openstack.org/v1beta1
+kind: WatcherDecisionEngine
+metadata:
+  finalizers:
+  - openstack.org/watcherdecisionengine
+  name: watcher-kuttl-decisionengine
```

Comment:
should be: 
`name: watcher-kuttl-decision-engine`

https://logserver.rdoproject.org/65/65/fe4aa769b44d1c038d93f638a11a82da2454d52c/github-check/watcher-operator-kuttl/968e37b/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/watcher-kuttl-default/crs/watcherdecisionengines.watcher.openstack.org/watcher-kuttl-decision-engine.yaml